docs/integrations/dev-setup.md

```diff
@@ -10,7 +10,13 @@ title: Development Setup
 ### Development OS
 Our recommended OS for development is either macOS or Linux, as we use bash and docker in some of our validation/testing flows.
 
-If you are working on Windows, you can either work with a Linux VM or utilize [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
+#### Windows Users
+If you are working on Windows, you can either work with a Linux VM or utilize [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10). 
+
+**Note:** When using WSL2 you may experience performance issues if working on the Windows mounted file system (for example `/mnt/c/`). See the following [WSL issue](https://github.com/microsoft/WSL/issues/4197) for more info. In such cases we recommend using the Linux file system (`ext4` partition) WSL2 provides. Meaning that the local demisto content and the SDK will all be located on the WSL file system And using an editor which supports remote WSL. Editors supporting remote WSL include:
```

Comment:
Added 
"Meaning that the local demisto content and the SDK will all be located on the WSL file system"